README.md

```diff
@@ -32,7 +32,7 @@ repositories {
 | 6. Heartbeat                     | `'live.ditto:dittoheartbeat:LIBRARY_VERSION'`              |
 | 7. Presence Degradation Reporter | `'live.ditto:presencedegradationreporter:LIBRARY_VERSION'` |
 
-The latest released version for all libraries is: `1.0.0`.
+The latest released version for all libraries is: `1.1.0`.
```

Comment:
I think it might simplify things to remove this line and have the single source of truth be the release from the link below which is marked as `Latest` in green

Reply:
im fine with that. PR updated.